crates/torii/graphql/src/tests/models_test.rs

```diff
@@ -15,40 +15,40 @@ mod tests {
             r#"
           {{
              recordModels {} {{
-              total_count
+              totalCount
               edges {{
                 cursor
                 node {{
                     __typename
-                    record_id
+                    recordId
```

Comment:
I think for model members we want to leave format casing as-is, otherwise there is naming conflict if someone intentionally deploys a model like below:

```
#[derive(Model, Copy, Drop, Serde)]
struct MyModel{
    mixed_case: u8,
    mixedCase: u8,
}
```